.circleci/config.yml

```diff
@@ -349,6 +349,10 @@ jobs:
           name: middleman build
           command: bundle exec middleman build
 
+      - run:
```

Comment:
If it makes things cleaner I can move this to its own PR. I was just experimenting with fixing the CI problems while @alvin-huang was clarifying some circle changes.